tests/python/frontend/onnx/test_forward.py

```diff
@@ -30,7 +30,8 @@
 import scipy
 
 
-def get_tvm_output(graph_def, input_data, target, ctx, output_shape=None, output_dtype='float32', opset=None):
+def get_tvm_output(graph_def, input_data, target, ctx, output_shape=None, output_dtype='float32',
+                   opset=None, mode=None):
```

Comment:
Just curious, why does `mode` need to be added? Does `NonZero` only work in the VM executor? If so, maybe we can just exclusively use the VM for testing, I'm not sure there's a need to support both.

Reply:
Yes, It makes sense and I will put it into another function instead of adding it to `get_tvm_output`.